### DISCUSSION TITLE
Adding "global" fields to every model

### DISCUSSION THREAD

**Discussion by ItsMeBrianD** [prisma/prisma#7233] [Q&A] [answered]
Hey everybody,

Been reading through the discussions and it looks like permutations of this have been asked, but the use case here is a little different. Previously we have used [MikroORM](https://mikro-orm.io/) for our database interactions, but there's a little too much "magic" happening behind the scenes and Prisma's approach looks like it is more clear. One thing that I haven't found a way to do cleanly with Prisma yet is to attach a set of fields to *every* model that we have, in our use case `created_at`, `created_by`, `updated_at`, `updated_by`, which are fairly common auditing fields.

Aside from adding them by hand to every model (and hoping nobody forgets in the future), is there a good solution for this yet?

**Comment by janpio** [comment, marked as answer]
No, nothing yet. There are multiple feature requests for different variants of that, but right now you best just add them manually or process your schema in some way to make sure they are present everywhere you need them.

**Reply by ItsMeBrianD** [reply]
Sad to hear, will keep my eye out for anything else that comes up
